data/schema/filters/abilities.cfg

```diff
@@ -10,7 +10,7 @@
 	{SIMPLE_KEY replacement_type string_list}
 	{SIMPLE_KEY alternative_type string_list}
 	{SIMPLE_KEY active_on ability_context}
-	{SIMPLE_KEY value s_int_range_list}
+	{SIMPLE_KEY value string_list}
```

Comment:
Would be nicer to add a type that accepts s_int_range_list and just "default".
Perhaps
```
[type]
    name=s_int_range_list_default
    [union]
        [type]
            link=s_int_range_list
        [/type]
        [type]
            value=default
        [/type]
    [/union]
[/type]
```
works.

Reply:
done